Further improvements to `Introduction to Elixir WebRTC` tutorial (#141)
elixir-webrtc/ex_webrtc: guides/introduction/forwarding.md

  WB((Web Browser)) <-.-> PC
```

The only thing we have to implement is the `Forwarder` process. In practice, making it a `GenServer` would be probably the
easiest and that's what we are going to do here. Let's combine the ideas from the previous section to write it.

```elixir
def init(_) do
  {:ok, pc} = PeerConnection.start_link(ice_servers: [%{urls: "stun:stun.l.google.com:19302"}])

>   # we expect to receive two tracks from the web browser - one for audio, one for video

Reviewer: Nitpick, to me this sounds better 

```suggestion
  # we expect to receive two tracks from the web browser - one audio and one video
```

>   # so we also need to add two tracks here, we will use these to forward media
>   # from each of the web browser tracks

Reviewer: Nitpick, I would write:

```suggestion
  # so we also need to add two tracks here, we will use them to loop media back to the browser
```
